chore: add `UNKNOWN` error code (#2562)
FuelLabs/fuels-ts: apps/docs/src/guide/errors/error-codes.md

The hashing algorithm is currently locked, any subsequent attempts to register a new implementation will throw this error.
The purpose of the lock function is to provide a way to ensure that the implementation of the specific hashing algorithm cannot be changed once it is locked. This can be useful in scenarios where you want to guarantee the integrity and consistency of the hashing function throughout your application.

> ## `UNKNOWN_ERROR`

Reviewer: You forgot to rename this.

```suggestion
## `UNKNOWN`
```

Author: Thanks!

> In cases where the error code is not known, this error will be thrown.

Reviewer: ```suggestion
In cases where the error hasn't been mapped yet, this code will be used.
```

> This may caused by a bug in the SDK or the Fuel Stack, please report the issue to the team.

Reviewer: ```suggestion
If you believe you found a bug, please report the [issue](https://github.com/FuelLabs/fuels-ts/issues/new/choose) to the team.
```
